### DISCUSSION TITLE
Version 8.4 beta 2

### DISCUSSION THREAD

**Discussion by Arnova8G2** [Arnova8G2/Gcam_8.7.250.44#65] [Ideas]
**Change logs :**

Version made to understand use of viewfinder to force super resolution or fixed freeze bug

**In developer**
Add camera.pixel.scene_brightness ( helps to fix samsung front cam crash, does not capture at moment)
Add camera.pixel.photo_optical_stabilization 
Add camera.pixel.format_viewfinder_stream ( viewfinder format ( 34 = PRIVATE ) by default )
Add camera.pixel.frames_viewfinder_stream ( viewfinder frames 9 by default )
Add camera.pixel.old_viewfinder_stream
Add camera.pixel.lowres_viewfinder_stream ( use resolution predefined by google )

<hr /><em>This discussion was created from the release <a href='https://github.com/Arnova8G2/Gcam_8.7.250.44/releases/tag/V8.4-Beta2'>Version 8.4 beta 2</a>.</em>

**Comment by myselflatu** [comment]
@Arnova8G2 HDR+ Control Is Missing From Night Sight 
![Screenshot_20230618-235606_Camera](https://github.com/Arnova8G2/Gcam_8.7.250.44/assets/40817763/d292d7a5-f830-4542-9a8f-af2c0c490152)

**Reply by Arnova8G2** [reply]
for hdr+ I do not add them for night, but add on portrait activate on dev ( camera.gouda.allow_hdrplus )

**Reply by myselflatu** [reply]
@Arnova8G2 oh okay

**Reply by Arnova8G2** [reply]
I can add it but you won't have good night light

**Reply by myselflatu** [reply]
@Arnova8G2 oh not needed then😅

**Comment by myselflatu** [comment]
@Arnova8G2 Can't Load Noise Model 
![Screenshot_20230619-114510_Camera](https://github.com/Arnova8G2/Gcam_8.7.250.44/assets/40817763/24cdcade-9d68-4734-8e88-0398ed249fc7)

**Reply by Arnova8G2** [reply]
download files using

![noise](https://github.com/Arnova8G2/Gcam_8.7.250.44/assets/40323300/4da4674f-03f9-4abb-a32d-745f3653da66)

**Reply by myselflatu** [reply]
@Arnova8G2 no that's not what i meant i have the file but importing it doesn't work

**Reply by Arnova8G2** [reply]
can you tell me what the file is or share it to me

**Comment by myselflatu** [comment]
@Arnova8G2 it was noise model of IMX520_PX4XL by CSTARK

**Reply by Arnova8G2** [reply]
@myselflatu , I found originals in my archives, test this
[CSTARK_noise.zip](https://github.com/Arnova8G2/Gcam_8.7.250.44/files/11792271/CSTARK_noise.zip)

**Reply by myselflatu** [reply]
@Arnova8G2 thanks this worked actually i use this noise model for Ultra Wide its kinda pretty aggressive with default one and gets blurry

**Reply by myselflatu** [reply]
@Arnova8G2 unfortunately I'm still getting blurry results even after applying the noise model ig it's not getting applied correctly 
![PXL_20230620_102719551](https://github.com/Arnova8G2/Gcam_8.7.250.44/assets/40817763/e0f41058-ef56-4c3a-8478-8af0a967478c)
As you can see from the sample above

**Reply by Arnova8G2** [reply]
fixed : 
https://github.com/Arnova8G2/Gcam_8.7.250.44/releases/tag/V8.4-Beta3
